saml2/saml2-service-provider/src/test/java/org/springframework/security/saml2/provider/service/metadata/OpenSamlMetadataResolverTests.java

```diff
@@ -78,4 +78,18 @@ public void resolveWhenRelyingPartyNoLogoutThenMetadataMatches() {
 		assertThat(metadata).doesNotContain("ResponseLocation");
 	}
 
+	@Test
+	public void shouldUseParametersConsumer() {
```

Comment:
At a matter of convention, will you please name this method so that it looks like `methodWhenConditionThenResult`?

For example, you might do `resolveWhenEntityDescriptorCustomizerThenUses`.